docs/topics/mta-rn-new-features.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features_{context}"]
+= New features
+
+This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
+
+.Application inventory module
+New application portfolio driven UI that enables organizations to manage and classify their applications using an extensible tagging model.
+
+.Applicational assessment module
```

Comment:
Spelling : Application not Applicational ?

Reply:
You are correct. Should be Application. Thank you!